python/graphscope/nx/tests/algorithms/builtin/test_simple_paths.py

```diff
@@ -0,0 +1,87 @@
+#
+# This file test_bfs.py is referred and derived from project NetworkX
+#
+# which has the following license:
+#
+# Copyright (C) 2004-2020, NetworkX Developers
+# Aric Hagberg <hagberg@lanl.gov>
+# Dan Schult <dschult@colgate.edu>
+# Pieter Swart <swart@lanl.gov>
+# All rights reserved.
+#
+# This file is part of NetworkX.
+#
+# NetworkX is distributed under a BSD license; see LICENSE.txt for more
+# information.
+#
+import pytest
+
+from graphscope import nx
+from graphscope.nx.tests.utils import almost_equal
+
+
+@pytest.mark.usefixtures("graphscope_session")
+class TestIsSimplePath:
+    """Unit tests for the
+    :func:`networkx.algorithms.simple_paths.is_simple_path` function.
+    """
+
+    def test_empty_list(self):
+        """Tests that the empty list is not a valid path, since there
+        should be a one-to-one correspondence between paths as lists of
+        nodes and paths as lists of edges.
+        """
+        G = nx.trivial_graph()
+        assert not nx.is_simple_path(G, [])
```

Comment:
we should call `nx.builtin.is_simple_path`, not `nx.is_simple_path`

Reply:
OK